src/pages/about.jsx

```diff
@@ -29,55 +36,160 @@ export const About = () => {
         bioacustics(AI Technology)
       </p>
 
-      <p className={aboutStyles.introduction}>
-        Orcasound is a coperative effort of many dedicated individuals and great
-        organizations.Here are our recent projects-- created by
-        volunteers,stewards,citizens,scientist,hackers and generous funders--
-        all working together for the orcas
-      </p>
+      <Box
+        sx={{
+          fontFamily: 'Mukta',
+          margin: '10px',
+        }}
+      >
+        <Container>
+          <Typography
+            mt={9}
+            align="justify"
+            variant="body1"
+            sx={{
+              fontFamily: 'Montserrat',
+            }}
+          >
+            Orcasound is a cooperative effort of many dedicated individuals and
+            great organizations. Here are our recent projects — created by
+            volunteers, stewards, citizen scientists, hackers, and generous
+            funders — all working together for the orcas.
+          </Typography>
 
-      <h2 className={aboutStyles.projects}>Our Projects</h2>
-      <div className={aboutStyles.card}>
-        <Card sx={{ maxWidth: 345, mb: 2 }}>
-          <CardMedia
-            component="img"
-            height="140"
-            image="https://www.aquariumcostadealmeria.com/wp-content/uploads/2019/01/shutterstock_554899423-1-1280x720.jpg"
-          />
-          <CardContent>
-            <Typography gutterBottom variant="h5" component="div">
-              Orcasound App
+          <Typography
+            mt={8}
+            mb={5}
+            variant="h4"
+            sx={{
+              fontFamily: 'Mukta',
+              fontWeight: '600',
+            }}
+          >
+            Our Projects
+          </Typography>
+
+          <Box>
+            <Grid
+              container
+              justifyContent="center"
+              spacing={mobileActive ? 0 : 5}
+            >
+              {Items.map((item) => {
+                return (
+                  <Grid item xs={12} sm={6} md={3} key={item.id}>
+                    {((mobileActive && item.id <= 2) ||
```

Comment:
It would be cleaner to have mobile and desktop code in their separate components. Something [like what paul ended up doing in the orcasite repo](https://github.com/orcasound/orcasite/blob/f745019967ef1f474ea0af7ad8c8cabab74ace7e/ui/src/components/Header.tsx#L20)

Reply:
In this case components for both mobile and desktop is same and it is just rendering based on some condition. Should I make like [this](https://github.com/orcasound/orcasite/blob/f745019967ef1f474ea0af7ad8c8cabab74ace7e/ui/src/components/Header.tsx#L20) ?